_delphi_utils_python/delphi_utils/validator/dynamic.py

```diff
@@ -79,7 +81,7 @@ def validate(self, all_frames, report):
         outlier_lookbehind = timedelta(days=14)
 
         # Authenticate API
-        covidcast.use_api_key(self.params.api_key)
+        # Epidata.auth = ("epidata", api)
```

Comment:
This can be removed.
```suggestion
```

Reply:
removed but are we not registering the call or because it's running in the same environment it won't be rate limited or just it's not something we need to worry about?

Reply:
You took care of adding the credentials at https://github.com/cmu-delphi/covidcast-indicators/pull/2004/files#diff-de79eb5faa25bf35238e1d85065a72bf9b7a264f92357b64b44b411423703f64R23

Also, this line was commented out, so it wasnt doing anything anyway.